entities/entities/spawned_money/cl_init.lua

```diff
@@ -3,27 +3,25 @@ include("shared.lua")
 function ENT:Initialize()
 end
 
+local vector_up = Vector(0,0,1)
```

Comment:
That's why I use self:OBBMaxs() - self:OBBCenter(), to get the local pos of the "highest" position of the model. Multiplying it by 0,0,1 simply isolates the z value while setting the x and y values to 0.